test/fixtures/parser/postgres/postgres_join_no_space.sql

```diff
@@ -0,0 +1,3 @@
+-- Not missing space before ON
+SELECT * FROM "my_table2"
+INNER JOIN "my_database"."my_schema"."my_table"ON ("my_table2".foo = "my_table".foo)

```

Comment:
Would you mind adding a yaml for this? https://github.com/sqlfluff/sqlfluff/tree/master/test/fixtures/parser